style/CONTRIBUTE.md

```diff
@@ -28,7 +28,7 @@ Pre-requisites:
   - **python3**, to run a local web server (`apt install python3`)
   - **nvm** (https://heynode.com/tutorial/install-nodejs-locally-nvm)
   - **xmlstarlet**, to preprocess imported SVGs for map displays (`apt install xmlstarlet`)
-  - **spritezero**, to combine icons into a stylesheet (`npm install -g @beyondtracks/spritezero-cli`)
+  - **spritezero**, to combine icons into a stylesheet (`npm install @beyondtracks/spritezero-cli`)
 
```

Comment:
Made these changes plus Makefile changes to master, recommend merging changes from main.